### PR TITLE
fix: remove unused setuptools import

### PR DESCRIPTION
**Pull request by pyrooka**
## PR summary
This commit removes an unused `setuptools` import from the `setup.py`
file. More details about the `setuptools` mess can be found in the following
PR: https://github.com/IBM/python-sdk-core/pull/203 

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,7 +14,6 @@
 # limitations under the License.
 
 from setuptools import setup
-from setuptools.command.test import test as TestCommand
```
Comment (padamstx) [inline setup.py:17]: Should we also remove the `tests_require`-related stuff?

**Review by padamstx — APPROVED** [review]
Looks good, but just had a question to be addressed before merging.

File: setup.py (hunk 1 above)
Comment (pyrooka) [reply setup.py:17]: We've discussed this here: https://github.com/IBM/python-sdk-core/pull/203#pullrequestreview-2205389145

**Pull request merged**